.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# To fully customize the contents of this image, use the following Dockerfile instead:
+# https://github.com/microsoft/vscode-dev-containers/tree/v0.128.0/containers/typescript-node-10/.devcontainer/Dockerfile
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:12
```

Comment:
I don't know what all is in here, but does it have C++ compilers?  This dockerfile installs cmake and ninja which I assume are for testing, but I didn't see any actual compilers.

Reply:
It has gcc. Didn't check where it comes from in detail though. But we can add it (+gdb) to this file anyway since they would just be no-op if they are already installed, and so we won't be confused if the base image is changed for some reason in the future.

Reply:
I like that idea